### PR TITLE
add triggerDisabled prop

### PR DESCRIPTION
**Pull request by williamboman**
In one of my use cases I wan't to simply disable the trigger handler (I let the component maintain all state internally so I'd like to avoid the need to implement my own trigger handler).

### CODE REVIEW

File: src/Collapsible.js [hunk 1]
```diff
@@ -191,6 +197,7 @@ var Collapsible = React.createClass({
     }
 
     var openClass = this.state.isClosed ? 'is-closed' : 'is-open';
+    var disabledClass = this.props.triggerDisabled ? 'is-disabled' : ''
```
Comment (williamboman) [inline src/Collapsible.js]: Would it be desired to have a modifier class if the trigger is not disabled?

Comment (glennflanagan) [inline src/Collapsible.js]: No, i think just one for disabled is good.

**Comment by williamboman** [comment]
@glennflanagan Do you want me to resolve conflicts or are you doing it rn?

**Comment by glennflanagan** [comment]
I'm on it.

**Comment by glennflanagan** [comment]
Good work 👍

**Pull request merged**